dbt/include/clickhouse/macros/materializations/snapshot.sql

```diff
@@ -123,14 +27,16 @@
     {% do return(tmp_relation) %}
 {% endmacro %}
 
-{% macro clickhouse__snapshot_merge_sql_one(target, source, insert_cols, upsert) -%}
+{% macro snapshot_merge_sql(target, source, insert_cols) -%}
```

Comment:
Good catch -- originally as you can see they had different signatures and at one point in my refactoring it seemed like the name `clickhouse__snapshot_merge_sql` didn't work.   I'll rename it back.  The separate macro is necessary because ClickHouse doesn't support `merge` at all.